backend/app/api/endpoints/adapter/task_members.py

```
@@ -26,13 +27,50 @@
+                title=title,
+                team_id=team_id,
+                team_name=team_name,
+                invited_by={
+                    "user_id": invited_by_user_id,
+                    "user_name": invited_by_user_name,
+                },
+            )
+            logger.debug(
+                f"Emitted task:invited event for user {user_id}, task {task_id}"
+            )
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)